docs/wiki/research/eODS.md

```diff
@@ -236,11 +252,11 @@ Regarding the enshrinement of liquid staking, Vitalik states:
 
 [^2]: https://notes.ethereum.org/@vbuterin/staking_2023_10#Protocol-and-staking-pool-changes-that-could-improve-decentralization-and-reduce-consensus-overhead
 
-[^3]: https://mirror.xyz/barnabe.eth/v7W2CsSVYW6I_9bbHFDqvqShQ6gTX3weAtwkaVAzAL4
+[^3]: https://mirror.xyz/Barnabé.eth/v7W2CsSVYW6I_9bbHFDqvqShQ6gTX3weAtwkaVAzAL4
```

Comment:
You might have found and replaced my name everywhere, now I don't know if that link will work ^^

Reply:
should be barnabe.eth, to be clear